targets/audio

```diff
@@ -138,16 +138,23 @@ void cras_sbc_codec_destroy(struct cras_audio_codec *codec) {
     abort();
 }
 END
+
+        # Find the cras test client
+        cras_test_client='tools/cras_test_client/cras_test_client.c'
+        if [ ! -f "$cras_test_client" ]; then
```

Comment:
Maybe add a note to remove this after 12334.0.0 (R77) becomes the minimum version we support.